cmd/sks_create.go

```diff
@@ -121,6 +123,9 @@ func (c *sksCreateCmd) cmdRun(_ *cobra.Command, _ []string) error { //nolint:goc
 		if c.NoMetricsServer {
 			delete(addOns, sksClusterAddonMetricsServer)
 		}
+		if c.ExoscaleCSI {
+			addOns[sksClusterAddonExoscaleCSI] = struct{}{}
```

Comment:
I feel like using a map here just over-complicates things.
This should be just three if that append to a slice.